src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1000,6 +1007,24 @@ class DocBaseViewer extends BaseViewer {
             // Add page IDs to each page after page structure is available
             this.setupPageIds();
         }
+
+        this.initThumbnails();
+    }
+
+    initThumbnails() {
+        this.thumbnailsSidebar.init({
+            totalItems: this.pdfViewer.pagesCount,
+            itemHeight: 115,
```

Comment:
Can we use the height of the first page with some defined lower limit as our starting value rather than hardcoding it?